src/battle/move/auto_jump_765C20.c

```diff
@@ -1,3 +1,34 @@
 #include "common.h"
 
+#ifdef NON_MATCHING
```

Comment:
Would be good to add a comment describing why this is non-matching.

Reply:
Same goes for all the nonmatching funcs - even something like "regalloc" or "float issues" would suffice